spec/event_sourcery/event_store/event_type_serializers/underscored_spec.rb

```diff
@@ -14,12 +14,24 @@
 
   describe '#deserialize' do
     it 'looks up the constant' do
-      expect(underscored.deserialize('ItemAdded')).to eq ItemAdded
-      expect(underscored.deserialize('ItemRemoved')).to eq ItemRemoved
+      expect(underscored.deserialize('item_added')).to eq ItemAdded
+      expect(underscored.deserialize('item_removed')).to eq ItemRemoved
```

Comment:
Looks like this worked because the implementation camelizes the input which is already camelized so it's a no-op.

https://github.com/envato/event_sourcery/blob/efcc13359166d0919248c2153c98d4753c800e49/lib/event_sourcery/event_store/event_type_serializers/underscored.rb#L54-L58